env/development

```diff
@@ -1,10 +1,10 @@
 PTERO_PETRI_AMQP_HOST=localhost
-PTERO_PETRI_AMQP_PORT=5672
 PTERO_PETRI_AMQP_VHOST=/
 PTERO_PETRI_AMQP_RETRY_DELAY=30
 PTERO_PETRI_AMQP_CONNECTION_ATTEMPTS=40
 PTERO_PETRI_AMQP_PREFETCH_COUNT=10
 PTERO_PETRI_AMQP_HEARTBEAT_INTERVAL=600
```

Comment:
Most of the environment variables in this file are not referenced anywhere else, which leads me to believe this file is way out of date and has gone out of use.  There also are no `env` sub-directories under the other service directories (i.e., `shell-command/env`, `workflow/env`).